src/main/java/seedu/utility/Ui.java

```diff
@@ -79,19 +79,13 @@ private void printLine() {
         System.out.println(" ");
     }
 
-    /**
-     * Prints the filtered list of expenses in the financial tracker to the standard output.
-     *
-     * @param expenses The list of expenses in the financial tracker.
-     */ 
-    public void listExpense(ArrayList<Expense> expenses) {
+    public void emptyList() {
         printLine();
         if (expenses.isEmpty()) {
             printEmptyExpenseListMessage();
         } else {
             printExpenseList(expenses);
```

Comment:
The result of this issue: https://github.com/AY2122S1-CS2113T-T12-3/tp/pull/49/files#r727998471